pom.xml

```
@@ -67,7 +67,7 @@
 
   <!-- Bring some sanity to version numbering... -->
   <properties>
-    <jenkins.version>2.138.4</jenkins.version>
+    <jenkins.version>2.222.1</jenkins.version>
```

I suspect this won't work, some plugins in the bom don't use the .x version,

```suggestion
    <jenkins.version>2.222.4</jenkins.version>
```